lib/hammer_cli/options/option_definition.rb

```diff
@@ -27,9 +27,9 @@ class OptionDefinition < Clamp::Option::Definition
       attr_accessor :deprecated_switches
 
       def initialize(switches, type, description, options = {})
-        self.value_formatter = options.delete(:format) || HammerCLI::Options::Normalizers::Default.new
-        self.context_target = options.delete(:context_target)
-        self.deprecated_switches = options.delete(:deprecated)
+        self.value_formatter = options[:format] || HammerCLI::Options::Normalizers::Default.new
```

Comment:
Is there a reason to remove the cleanup of the options?

Reply:
If I leave the cleanup then the options hash for the `option` in predefined options will be empty, which makes the predefined option unusable for other commands.